cloud.deploy.yaml

```diff
@@ -1,10 +1,4 @@
 steps:
-  - name: "gcr.io/cloud-builders/npm"
-    args: ["install"]
-  - name: "gcr.io/cloud-builders/npm"
-    args: ["run", "test"]
-  - name: "gcr.io/cloud-builders/npm"
-    args: ["run", "build"]
```

Comment:
Likely controversial. My reasoning is that QA should run and block merger to protected branches (master in the case). At that point we should trust it and merely view prod deployments as promotions, and should't feel the need to run tests/linters etc again

Let's discuss :)